index.html

```diff
@@ -1,48 +1,297 @@
+            </ul>
 
 
- </section>
- </main>
+        </section>
+
+    
+    
+    </main>
+
  
     
-</body>
+        <header>
+
+            <div><button  class="scroll-down" type="button">
+                <i class="fa fa-angle-down fa-2x"></i>
+            </button></div>
+
+            <div class="recent-works"><h1>My Recent Works</h1></div>
+            <div class="indicator">
+                <img alt="indcator" src="img/Indicator.png">
+            </div>       
+
+        </header>
+    <div class="wrapper">
+
+        <div>
+            <section id="works1" class="container3">
+
+
+                <div class="item3"><img alt="work1" src="img/work1.png"></div>
+
+                
+                            
+                <div class="item3"><h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                <div class=" item3 btn-group">
+                    <button class="languages"> Ruby on rails</button>
+                    <button class="languages"> css</button>
+                    <button class="languages"> JavaScript</button>
+                    <button class="languages"> html</button>
+                </div>
+
+                
+                    <div class="item3">
+                        <button class="see-project">See project</button></div>
+                
+                
+
+
+
+            </section>
+        </div>
+
+        <div>
+            <section id="works2" class="container4">
+
+
+                <div class="item4"><img alt="work2" src="img/work2.png"></div>
+
+                
+                            
+                <div class="item4"><h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                <div class=" item4 btn-group">
+                    <button class="languages"> Ruby on rails</button>
+                    <button class="languages"> css</button>
+                    <button class="languages"> JavaScript</button>
+                    <button class="languages"> html</button>
+                </div>
+
+                
+                <div class="item4"><button class="see-project">See project</button></div>
+                
+                
+
+
+
+            </section>
+        </div>
+
+        <div>
+                <section id="works3" class="container5">
+
+
+                    <div class="item5"><img alt="work3" src="img/work3.png"> </div>
+
+                    
+                                
+                    <div class="item5"><h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                    <div class=" item5 btn-group">
+                        <button class="languages"> Ruby on rails</button>
+                        <button class="languages"> css</button>
+                        <button class="languages"> JavaScript</button>
+                        <button class="languages"> html</button>
+                    </div>
+
+                    
+                    <div  class="item5"><button class="see-project">See project</button></div>
```

Comment:
- [ ] The button (e.g., "See project") is a link a or button tag nice it is a button, and  you followed the button styling: Active, Hover and Pressed, however you didn't add the `hover color` as well as add `cursor: pointer`